modules/rosa-policy-security-and-compliance.adoc

```diff
@@ -0,0 +1,125 @@
+//Modules included in the following assemblies:
+//
+// * rosa_architecture/rosa_policy_service_definition/rosa-policy-shared-responsibility.adoc
+
+[id="rosa-policy-security-compliance_{context}"]
+= Security and regulation compliance
+The following table outlines the  the responsibilities in regards to security and regulation compliance:
+
+[cols="2a,3a,3a",options="header"]
+|===
+
+|Resource
+|Service responsibilities
+|Customer responsibilities
+
+|Logging
+|**Red Hat**
+
+- Send cluster audit logs to a Red Hat SIEM to analyze for security events. Retain audit logs for a defined period of time to support forensic analysis.
+|- Analyze application logs for security events.
+- Send application logs to an external endpoint through logging sidecar containers or third-party logging applications if longer retention is required than is offered by the default logging stack.
+
+|Virtual networking management
+|**Red Hat**
+
+- Monitor virtual networking components for potential issues and security threats.
+
+- Use public AWS tools for additional monitoring and protection.
+
+|- Monitor optional configured virtual networking components for potential issues and security threats.
+- Configure any necessary firewall rules or customer data center protections as required.
+
+|Virtual storage management
+|**Red Hat**
+
+- Monitor virtual storage components for potential issues and security threats.
+
+- Use public AWS tools for additional monitoring and protection.
+
+- Configure the ROSA service to encrypt control plane, infrastructure, and worker node volume data by default using the
+AWS managed Key Management Service (KMS) key that Amazon EBS provides.
+
+- Configure the ROSA service to encrypt customer persistent volumes that use the default storage class with the AWS
+managed KMS key that Amazon EBS provides.
+
+- Provide the ability for the customer to use a customer managed AWS KMS key to encrypt persistent volumes.
```

Comment:
ROSA can encrypt etcd with customer managed KMS keys as well as persistent volumes, right?

Reply:
I am going to make a separate ticket to address these additions.